app/views/catalog/_home_text.html.erb

```diff
@@ -3,6 +3,7 @@
   <%= link_to 'check out our discovery roadmap', 'https://libraries.psu.edu/library-search-roadmap' %>.</p>
 <p>You can also:</p>
 <ul>
+  <li><%= link_to 'Search the legacy CAT', 'https://cat.libraries.psu.edu' %></li>
```

Comment:
I wonder if it would be better to just say "Search The CAT (legacy system)" (or something) just to avoid folks calling the new catalog "the new cat"? Thinking out loud really.

"Search The CAT (legacy)"?
"Search The CAT (legacy catalog)"?  
Or just "Search The CAT"?

🤔

Reply:
I personally liked "Search the CAT (legacy catalog)"

Reply:
I tried making that change but it needs review bloew?